src/settings/DataExportSettings/DataExportSettings.js

```diff
@@ -3,10 +3,40 @@ import { FormattedMessage } from 'react-intl';
 
 import { Settings } from '@folio/stripes/smart-components';
 import { stripesShape } from '@folio/stripes/core';
+import {
+  ProfilesLabel,
+  SettingsLabel,
+} from '@folio/stripes-data-transfer-components';
 
-export function DataExportSettings(props) {
-  const sections = [];
+import { MappingProfiles } from '../MappingProfiles';
 
+const getSettingsLabel = (messageId, iconKey) => {
+  return (
+    <SettingsLabel
+      messageId={`ui-data-export.${messageId}`}
+      iconKey={iconKey}
+      app="data-export"
+    />
+  );
+};
+
+const sections = [
+  {
+    label: <ProfilesLabel
+      link="https://wiki.folio.org/display/FOLIOtips/Creating+and+using+profiles"
```

Comment:
As I see this link is used on data-import. I am not sure that in our case the link would be the same. I would suggest removing a link or not providing it at all. What do you think?

Reply:
I think it is used to the information about creating and using profile that will be the same in case of ui-data-import and ui-data-export. So I suggest using this link and inform about it our PO. It will be easy to change it in the future.

Reply:
Yeah with informing PO sounds good.